html-aam/index.html

```diff
@@ -7951,6 +7951,9 @@ <h3>HTML Attribute State and Property Mappings</h3>
             </div>
           </li>
           <li>All elements having an accessible object in IAccessible2 mapping are supposed to implement IAccessible, IAccessible2 and IAccessible2_2 interfaces.</li>
+          <li>
+            User interaction with certain HTML elements will set an element's <a data-cite="html/form-control-infrastructure.html#concept-fe-dirty">dirty value flag</a>. For these elements which are identified in [HTML], if specified such attributes represent only the default value or state for the element. 
```

Comment:
I think I don't understand the second sentence of this.. specifically:

> if specified such attributes represent only the default value or state of the element.

Is it me, or could it be made more clear?